### ISSUE TITLE
Request for the Continue training. And Updated version have dependency error,

### ISSUE THREAD

**Issue by silverline-art** [paninski-lab/lightning-pose#290]
Greetings,
The Problem: Currently there's no option to continue training of the already trained model or fine tuning the model with the updated dataset.
Please add this feature in next update please. For the Pose application.

Dependency Issue:
For the Updated version of the app

Installing collected packages: lightning
  Attempting uninstall: lightning
    Found existing installation: lightning 2.2.5
    Uninstalling lightning-2.2.5:
      Successfully uninstalled lightning-2.2.5
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
lightning-pose-app 1.5.0 requires lightning[app]==2.2.5, but you have lightning 2.5.1.post0 which is incompatible.
Successfully installed lightning-2.5.1.post0
/home/shivam/Desktop/Python_Venvs/lia/bin/python3.10: No module named lightning.__main__; 'lightning' is a package and cannot be directly executed

**Comment by themattinthehatt** [comment]
Hi @silverline-art thanks for the issue. I'll look into continuing model training and get back to you.

With regards to updating the dataset/adding more labels, we've found that it's usually better to just train the model from scratch using the full dataset, rather than finetuning on the labels.

For the dependency issue, please make sure to *not* upgrade your lightning version whenever you launch the app. You'll need to uninstall your current version of lightning and reinstall the old version for the app to work; please see the FAQ "I accidentally upgraded my lightning version, what should I do?" here: https://pose-app.readthedocs.io/en/latest/source/faqs.html

**Comment by silverline-art** [comment]
Thank you for your response:
For continuing model training;  It is for fine-tune you can say I think.
2 cases:
Case 1: As I usually try to test both Semi-/Supervised Model with different numbers of frames in datasets, like 20 frames, 100 frames 300 frames.
Case 2: I'm also interested training the existing model with new dataset of similar in nature. for the purpose of fine-tuning and making the model more robust.
